R/bestlag.R

```diff
@@ -1,6 +1,6 @@
 #' Best Lag
 #'
-#' This function computes the best lags for a dataset using weighted correlation
+#' This function computes the best lags for a dataset using weighted correlation. The lags obtained are in reference to the original timepoints.  
```

Comment:
Can you switch this to one sentence per line?

Reply:
Done.